### ISSUE TITLE
Add support for vitest

### ISSUE THREAD

**Issue by jasonkuhrt** [prisma-labs/konn#30]
#### Perceived Problem

- Jest is not maintained anymore

#### Ideas / Proposed Solution(s)

- Add support for Vitest https://vitest.dev/api/#setup-and-teardown

**Comment by juliusmarminge** [comment]
still considering this?

**Comment by jasonkuhrt** [comment]
@juliusmarminge yep! Actually we're trying to switch to vitest at Prisma so this will become more important for us in the coming months.


After using vitest on a few side projects now I feel pretty confident that this will be easy in that its not so different than Jest.

**Comment by juliusmarminge** [comment]
Great I'd love to switch tRPC over to Vitest too but kinda blocked by this

**Comment by jasonkuhrt** [comment]
@juliusmarminge it helps to know others would benefit from this. I'll try to get some work time to do this in the coming weeks.

**Comment by juliusmarminge** [comment]
> @juliusmarminge it helps to know others would benefit from this. I'll try to get some work time to do this in the coming weeks.

Hey, how's it looking like?

**Comment by jasonkuhrt** [comment]
Getting work time for this is hard with current priorities. I will ask about getting this project's ownership given to me personally where then I can keep chipping away at it with my personal investment. Not sure if that will happen. I'll be asking my manager after January 8.

Thanks for your continued interest!
